views/layouts/default.hbs

```diff
@@ -42,21 +42,21 @@
     <footer id="footer">
       <div class="section-wrapper">
         <div class="third">
-          <a class="footer-logo-wrapper" href="https://www.mozilla.org" target="_blank" rel="noopener"><img src="/img/moz-logo-bw-rgb.svg" alt="Mozilla Logo" /></a>
+          <a class="footer-logo-wrapper" href="https://www.mozilla.org?utm_source={{ SERVER_URL }}&utm_medium=referral&utm_campaign=fx_monitor_downloads&utm_content=footer-link" target="_blank" rel="noopener"><img src="/img/moz-logo-bw-rgb.svg" alt="Mozilla Logo" /></a>
         </div>
         <ul class="third">
           <li>Support</li>
           <li><a href="https://qsurvey.mozilla.com/s3/Firefox-Monitor-Feedback" target="_blank" rel="noopener">Give Feedback</a></li>
-          <li><a href="https://www.mozilla.org/about/legal/" target="_blank" rel="noopener">Terms</a></li>
-          <li><a href="https://www.mozilla.org/privacy/websites/" target="_blank" rel="noopener">Privacy</a></li>
+          <li><a href="https://www.mozilla.org/about/legal/?utm_source={{ SERVER_URL }}&utm_medium=referral&utm_campaign=fx_monitor_downloads&utm_content=footer-link" target="_blank" rel="noopener">Terms</a></li>
+          <li><a href="https://www.mozilla.org/privacy/websites/?utm_source={{ SERVER_URL }}&utm_medium=referral&utm_campaign=fx_monitor_downloads&utm_content=footer-link" target="_blank" rel="noopener">Privacy</a></li>
```

Comment:
Looking at this (versus one of the email ones above) side-by-side, I'm a bit confused between `content=email_footer` versus `utm_content=footer-link`.

Both in the consistency of `email_footer` versus `footer-link` (underscores versus hyphens) as well as `content=...` vs `utm_content=...`:

```
https://www.mozilla.org/privacy/websites/?utm_source=email&utm_medium=email&utm_campaign=fx_monitor_downloads&content=email_footer
https://www.mozilla.org/privacy/websites/?utm_source={{ SERVER_URL }}&utm_medium=referral&utm_campaign=fx_monitor_downloads&utm_content=footer-link
```

| SITE | `utm_source` | `utm_medium` | `utm_campaign` | `utm_content`*
|------|--------------|--------------|----------------|----------------
| https://www.mozilla.org/privacy/websites/ | email | mail | fx_monitor_downloads | content=email_footer
| https://www.mozilla.org/privacy/websites/ | `{{ SERVER_URL }}` | referral | fx_monitor_downloads | utm_content=footer-link